vector/src/main/res/layout/item_space_directory_filter_no_results.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.appcompat.widget.LinearLayoutCompat xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think it's not necessary to explicitly use `LinearLayoutCompat`, `LinearLayout` is OK. I think `LinearLayoutCompat` will be inflated at runtime by `AppCompatViewInflater` anyway (but after checking this file I may be wrong).